edge-node-container/.trivyignore

```diff
@@ -17,3 +17,18 @@ AVD-KSV-0118
 # enic-utils issues, will be addresses post-3.0
 CVE-2025-31115
 CVE-2024-56406
+
```

Comment:
The CVEs are traced back to latest base image, so, okay to proceed. Let's revisit in next release and update this list to check if the same CVEs are still there and plan an action.